contrib/src/test/java/com/datatorrent/contrib/enrich/FileEnrichmentTest.java

```diff
@@ -165,5 +165,75 @@ public void testEnrichmentOperatorDelimitedFSLoader() throws IOException, Interr
     Assert.assertEquals("value of year", 2016, mfgDate.getYear() + 1900);
     Assert.assertTrue(emitted.get("mfgDate") instanceof Date);
   }
+  
+  @Test
+  public void testEnrichmentOperatorFixedWidthFSLoader() throws IOException, InterruptedException
+  {
+    URL origUrl = this.getClass().getResource("/fixed-width-sample.txt");
+
+    URL fileUrl = new URL(this.getClass().getResource("/").toString() + "fixed-width-sample1.txt");
+    FileUtils.deleteQuietly(new File(fileUrl.getPath()));
+    FileUtils.copyFile(new File(origUrl.getPath()), new File(fileUrl.getPath()));
```

Comment:
Why do you need to copy and then set the new file path to FSLoader? You could just use origUrl object.